src/pages/404/error.js

```diff
@@ -0,0 +1,15 @@
+import "./error.css"
+function Error(){
```

Comment:
We can name this component a little better. As `error.js` is somewhat confusing. We can also name it `NotFound.js`.